db_stress_tool/db_stress_test_base.cc

```diff
@@ -2883,6 +2883,24 @@ void InitializeOptionsFromFlags(
   options.blob_compaction_readahead_size = FLAGS_blob_compaction_readahead_size;
   options.blob_file_starting_level = FLAGS_blob_file_starting_level;
 
+  if (FLAGS_use_blob_cache) {
```

Comment:
One more thing: it might make sense to print some info about the cache configuration as part of this `printf`: https://github.com/facebook/rocksdb/blob/main/db_stress_tool/db_stress_test_base.cc#L2322-L2335

Reply:
Done. Thanks.